mergin/client_pull.py

```diff
@@ -694,6 +719,8 @@ def download_file_finalize(job):
     # make sure any exceptions from threads are not lost
     for future in job.futures:
         if future.exception() is not None:
+            job.mp.log.error("Error while downloading file: " + str(future.exception()))
+            job.mp.log.info("--- download aborted")
```

Comment:
I would suggest to remove this change, download_file is used separately and got replaced by download_files implementation,,,